src/assisted_installer_controller/assisted_installer_controller_test.go

```diff
@@ -1927,38 +1924,14 @@ var _ = Describe("installer HostRoleMaster role", func() {
 
 			mockk8sclient.EXPECT().GetClusterVersion().Return(clusterVersionReport, nil).AnyTimes()
 			mockbmclient.EXPECT().GetClusterMonitoredOperator(gomock.Any(), gomock.Any(), cvoOperatorName, gomock.Any()).Return(currentServiceCVOStatus, nil).AnyTimes()
-			mockbmclient.EXPECT().UpdateClusterOperator(gomock.Any(), gomock.Any(), cvoOperatorName, "", gomock.Any(), gomock.Any()).AnyTimes()
+			mockbmclient.EXPECT().UpdateClusterOperator(gomock.Any(), gomock.Any(), cvoOperatorName, "", gomock.Any(), gomock.Any()).Return(fmt.Errorf("dummy")).AnyTimes()
 
 			err := func() error {
 				return assistedController.waitingForClusterOperators(ctx)
 			}()
 
 			Expect(errors.Is(err, context.DeadlineExceeded)).To(BeTrue())
 		})
-
-		It("service fail to sync - finally succeed", func() {
```

Comment:
Why did you remove this test?

Reply:
Cause this test case actually tested that we do one more loop and not returning ready immediately